packages/stats/gbstats/models/tests.py

```diff
@@ -15,8 +15,9 @@
 @dataclass
 class BaseConfig:
     difference_type: DifferenceType = "relative"
-    traffic_proportion_b: float = 1
+    traffic_percentage: float = 1
     phase_length_days: float = 1
+    total_users: int = 0
```

Comment:
Make the default here None and make it optional so we can properly throw errors.

Reply:
done